src/scene/graph-node.js

```diff
@@ -1099,6 +1099,17 @@ class GraphNode extends EventHandler {
     }
     // #endif
 
+    // fires an event on all children of the node
+    _fireOnHierarchy(name) {
+        if (this.fire) {
```

Comment:
Any idea why there is a check for 'fire' here? GraphNode inherits from EventHandler which defines fire function and can't really see when that would be missing.

Reply:
good point and no idea. I removed it and all seems to still work :)